src/constants/index.js

```diff
@@ -12,6 +12,11 @@ const syncStatus = {
     HAS_ERROR: 'hasError'
 };
 
+const surveyList = [
+    {idGroup: surveyState.OPENED, tabName: 'Pendientes'},
```

Comment:
la constante debería ser `surveyAddressState` y debería ser la misma que acá: https://github.com/indec-it/sivy/blob/master/src/model/surveyAddressState.js

Reply:
Listo